### PR TITLE
remove redundant warning

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: pkg/sysinfo/sysinfo.go [hunk 1]
```diff
@@ -42,14 +42,13 @@ func New(quiet bool) *SysInfo {
...
-			logrus.Warnf("WARING: %s\n", err)
+			logrus.Warnf("%s\n", err)
 		}
 	} else {
 		_, err1 := ioutil.ReadFile(path.Join(cgroupCpuMountpoint, "cpu.cfs_quota_us"))
-		logrus.Warnf("%s", cgroupCpuMountpoint)
 		sysInfo.CpuCfsQuota = err1 == nil
 		if !sysInfo.CpuCfsQuota && !quiet {
-			logrus.Warnf("WARING: Your kernel does not support cgroup cfs quotas")
+			logrus.Warnf("Your kernel does not support cgroup cfs quotas")
```
Comment (RickWieman) [inline pkg/sysinfo/sysinfo.go]: This can be `logrus.Warn`, right?

File: pkg/sysinfo/sysinfo.go [hunk 2]
```diff
@@ -42,14 +42,13 @@ func New(quiet bool) *SysInfo {
 
 	if cgroupCpuMountpoint, err := cgroups.FindCgroupMountpoint("cpu"); err != nil {
 		if !quiet {
-			logrus.Warnf("WARING: %s\n", err)
+			logrus.Warnf("%s\n", err)
```
Comment (LK4D4) [inline pkg/sysinfo/sysinfo.go]: I think `\n` not necessary too. Also format should be `%v`

**Comment by hqhq** [comment]
Updated.

**Comment by LK4D4** [comment]
Hmm, still failing. I'll fix it.

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**